content/articles/domain-apex-heroku.markdown

```diff
@@ -1,33 +1,63 @@
 ---
 title: Pointing the Domain Apex to Heroku
-excerpt: Welcome to DNSimple. This page is about pointing the domain apex at Heroku. Hosted DNS has never been this easy.
+excerpt: How to point your domain to your Heroku application using DNSimple.
 categories:
 - Heroku and DNSimple
 ---
 
 # Pointing the Domain Apex to Heroku
 
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
 Your domain apex is the second or third level domain provided to you by your domain registrar. For example, `dnsimple.com` is the apex domain for DNSimple.
 
-To point your apex to Heroku you have 2 possibilities:
+To point your apex to Heroku you have 2 options:
 
-- Use the Heroku one-click service (for non-SSL usage)
-- Add an [ALIAS record](/articles/alias-record/) pointing your custom Heroku end point (for non-SSL and SSL usage)
+1. Using the Heroku one-click service.
+1. Manually adding an [ALIAS record](/articles/alias-record/).
```

Comment:
We may need some :us: :person_with_blond_hair: here, but I prefer the non-ing version

- Use ...
- Manually add...